glusterd2/volume/struct.go

```diff
@@ -151,11 +152,13 @@ func NewBrickEntries(bricks []api.BrickReq, volName, volfileID string, volID uui
 			return nil, e
 		}
 
-		switch b.Type {
+		switch strings.ToLower(b.Type) {
```

Comment:
`strings.ToLower` shouldn't be needed. The request validation phase should catch and reject. The API shouldn't accept "Arbiter" but only "arbiter"